week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -1,34 +1,35 @@
-/*
-  The space travellers have safely landed and are foraging for food in the natural wildlife.
-
-  There are bushes with many different colour berries.
-  The pink berries are the ONLY safe ones to eat.
-  If any other berries are present, it's best not to eat from the bush at all!
-
-  Create a function which checks if the bush has ALL PINK berries and is safe for the astronauts to eat from the bush.
-
-  Use the tests to confirm which message to return
-*/
-
-function bushChecker() {
-
+// this function checks if every string in the array is pink then bush is safe otherwise it is toxic.
+
+function bushChecker(bushBerries) {
+  let bushBerryColours = bushBerries.every(
```

Comment:
Its good to have a variable name that reflects what is inside it - maybe a better name for this would be "bushContainsPinkBerries"?